src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -148,22 +148,19 @@ class NotebooksContent extends Component {
   async uploadFiles(files) {
     const { namespace, workspace: { workspace: { bucketName } } } = this.props
     const existingNames = this.getExistingNames()
-    try {
-      this.setState({ saving: true })
-      await Promise.all(_.map(async file => {
-        const name = file.name.slice(0, -6)
-        if (_.includes(name, existingNames)) {
-          throw new Error(`${name} already exists`)
-        }
-        const contents = await Utils.readFileAsText(file)
-        return Buckets.notebook(namespace, bucketName, name).create(JSON.parse(contents))
-      }, files))
-      this.refresh()
-    } catch (error) {
-      reportError('Error creating notebook', error)
-    } finally {
```

Comment:
What's the reason for removing the error handling?

Reply:
Hmm, okay. I defer to Isaac on that then.

Reply:
JK, added error handling back in case something goes wrong with other aspects of the body of code.